config/routes.rb

```diff
@@ -138,13 +140,11 @@
               put :withdraw
 
               scope module: :participants do
-                resources :outcomes, only: %i[create index], as: :participant_outcomes
+                resources :outcomes, only: %i[create index], as: :participants_outcomes
```

Comment:
So I think we need both lists one for a single participant and one for multiple, the decision to separate those was intentional:
- we return a list of all outcomes for all declarations
- we return a list of all outcomes for a single participant
- we create an outcome for a participant
happy to chat through